### ISSUE TITLE
Transfer learning results in paper

### ISSUE THREAD

**Issue by gfotedar** [zhanghang1989/ResNeSt#148]
For the segmentation results given in the paper, did you pretrain on imagenet?

**Comment by zhanghang1989** [comment]
Yes. Please see the link to reproduce segmentation results https://hangzhang.org/PyTorch-Encoding/model_zoo/segmentation.html
